app/ts/schema.ts

```diff
@@ -18,12 +17,9 @@ export function createCacheParser<T>(funType: funtypes.Codec<T>) {
 	return config
 }
 
-export const AddressSchema = funtypes.String.withGuard(isHexString).withConstraint(isAddress, { name: 'Address' })
-export type Address = funtypes.Static<typeof AddressSchema>
-
-export const BigIntParser: ParsedValueConfig<funtypes.String, bigint> = {
+export const BigIntParser: funtypes.ParsedValue<funtypes.String, bigint>['config'] = {
 	parse(value) {
-		if (!isHexString(value)) return { success: false, message: `${value} is not a hex string encoded number.` }
+		if (!/^0x([a-fA-F0-9]{1,64})$/.test(value)) return { success: false, message: `${value} is not a hex string encoded number.` }
```

Comment:
This doesn't seem to be correct, bigints can be bigger than 64 numbers

Reply:
this is just a copy of wire-types implementation [here](https://github.com/DarkFlorist/TheInterceptor/blob/8eb9bda8cf3d19e100394516a14f5bd96b533d17/app/ts/types/wire-types.ts#L4). Any suggestion of a better test?